Formula/uftp.rb

```diff
@@ -0,0 +1,53 @@
+class Uftp < Formula
+  desc "secure, reliable, efficient multicast file transfer program"
+  homepage "http://uftp-multicast.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/uftp-multicast/source-tar/uftp-4.9.3.tar.gz"
+  sha256 "9e9215af0315257c6cc4f40fbc6161057e861be1fff10a38a5564f699e99c78f"
+
+  depends_on "openssl"
+
+  def install
+    system "make", "OPENSSL=#{Formula["openssl"].opt_prefix}", "DESTDIR=#{prefix}", "install"
+    # the makefile installs into DESTDIR/usr/..., move everything up one and remove usr
```

Comment:
re: source browser, I don't see one - he hosts release files on sourceforge, but there doesn't seem to be an svn repository behind it.

Here's the (brief) exchange:
 
Dan,

This is one of those things that each OS has their own take on.  Some prefer /bin which others prefer /usr/bin.  I went with /usr/bin as most Linux distributions are set up this way, and Debian in particular requested it.  So unfortunately the makefile will need to stay as it is.

Regards,
Dennis

On Sun, Feb 12, 2017 at 1:07 PM, <dan_berger@users.sf.net> wrote:

> Hi Dennis;
> 
> I've written a formula to package uftp for brew (https://brew.sh/)
> 
> In doing so, I had to work around the fact that the install paths in the makefile append usr to DESTDIR.
> Please consider removing the baked in usr path component, so one could (for example) say DESTDIR=/opt/uftp and end up with /opt/uftp/bin/uftp rather than /opt/uftp/usr/bin/...
> 
> Cheers
> 
> ...Dan